Remove Full Track Name (#331)
moq-wg/moq-transport: draft-ietf-moq-transport.md

with it.  A track name identifies an individual track within the
namespace.

A MOQT scope is a set of servers (as identified by their connection
URIs) for which the tuple of Track Name and Track Namespace are
guaranteed to be unique and identify a specific track. It is up to
the application using MOQT to define how broad or narrow the scope is.
An application that deals with connections between devices
on a local network may limit the scope to a single connection; by
contrast, an application that uses multiple CDNs to serve media may
require the scope to include all of those CDNs.

> The tuple of Track Namespace and Track Name identify the track
> within a given MOQT scope and is used as cache key.

Author: ```suggestion
Because the tuple of Track Namespace and Track Name are unique within an
MOQT scope, they can be used as a cache key.
```

with it.  A track name identifies an individual track within the
namespace.

A MOQT scope is a set of servers (as identified by their connection
URIs) for which the tuple of Track Name and Track Namespace are
guaranteed to be unique. Within a single MOQT scope, subscribing to the

> same Track Name and Track Namespace results in the subscriber receiving

Reviewer: s/  subscribing to the same Track Name and Track Namespace are guaranteed/  

  ==> subscribing to the same tuple of Track Name and its Namespace results in ...

It is important to know the combination of these 2 (as a tuple) is what identifies a track and just track name, since they are no unique even within a moq scope

Author: Thanks Suhas, I clarified this and shortened the text.
